### DISCUSSION TITLE
GAMS:        'gams' executable not in PATH

### DISCUSSION THREAD

**Discussion by ltambwe** [iiasa/message_ix#866] [MESSAGE modeling]
I am getting the error: GAMS:        'gams' executable not in PATH
Have:
- gams.exe  in "C:\GAMS\47"
- The PATH is in the environment variables and is correctly reported when I issue 'echo'
Any comments would be appreciated.

**Comment by glatterf42** [comment]
Hi @ltambwe, could you please provide some more details? For starters, when you run `message-ix show-versions` in your terminal, what is the output that you see?
Also, what exactly is the command you're trying to run that leads to the error you report?
